docs/guides/deploy-guide/bootstrap.md

```diff
@@ -102,53 +152,56 @@ the nodes are already ready for use.
    * System is currently rebooting and is not yet accessible via network.
 
      ```
-     fatal: [net003]: UNREACHABLE! => {"changed": false, "msg": "Connection timed
-     out.", "unreachable": true}``
+fatal: [net003]: UNREACHABLE! =>
```

Comment:
Why is this necessary?

Reply:
The build process handles this as some kind of javascript snippet: https://github.com/osism/osism.github.io/actions/runs/9548256571/job/26315015614

See "message": "Could not parse expression with acorn"

It seemed that it went away, bug now the problem is back :-(
Reverted now the example output to the original output.
I currently have no good idea what to do to make it work again :-)